core/src/main/scala/com/evolutiongaming/catshelper/ParallelHelper.scala

```diff
@@ -2,11 +2,26 @@ package com.evolutiongaming.catshelper
 
 import cats.{Foldable, Monoid, Parallel}
 
-object ParallelHelper {
+object ParallelHelper extends ParallelSyntax
+
+trait ParallelSyntax {
```

Comment:
why not move all `*Syntax` classes into own files?

Reply:
also I propose to make alternative  `object *Syntax` and leave `Helper` for for backward compatibility